samples/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath("com.android.tools.build:gradle:3.2.1")
+        classpath(kotlin("gradle-plugin", version = "1.3.20"))
+        classpath("org.mozilla.android:gradle-plugin:+") // Substituted in.
```

Comment:
nit: add a comment that getting this dependency from plugins.mozilla.org is different from what we're doing here (which is pulling it locally). e.g. we need to add the "plugins.mozilla.org" repository for the `apply` syntax to work.